README.md

```diff
@@ -8,23 +8,73 @@
 [![Build Status](https://travis-ci.com/gcivil-nyu-org/spring2020-cs-gy-9223-class.svg?branch=master&service=github)](https://travis-ci.com/gcivil-nyu-org/spring2020-cs-gy-9223-class)
 [![Coverage Status](https://coveralls.io/repos/github/gcivil-nyu-org/spring2020-cs-gy-9223-class/badge.svg?branch=master)](https://coveralls.io/github/gcivil-nyu-org/spring2020-cs-gy-9223-class?branch=master&service=github)
 
-# First time repo setup
-1. From the root of the repo, run `scripts/setup.sh`. Activate your virtualenv first.
-2. To run the server, run `python manage.py runserver`. You should now be able to view the site at http://127.0.0.1:8000/
-3. To setup SymmetricsDs on your machine, click [here.](https://github.com/gcivil-nyu-org/spring2020-cs-gy-9223-class/blob/master/symmetricds/README.md) In case you are on windows, please also refer to the wiki on the topic by going [here.](https://github.com/gcivil-nyu-org/spring2020-cs-gy-9223-class/wiki/Setting-up-SymmetricDS-on-Windows) Windows support is still not streamlined but this should be enough to get you started. In case you want to help us make this better please refer below as to how you can contribute to the project. 
+# Requirements
+- python 3.6 or above
+- In case of setting up synchronization, no whitespace is allowed in the path to the project directory.
 
+# Setup
+### For Ubuntu,
+1. Do
+```
+sudo apt install python3-dev libpq-dev
+```
+
+2. Install the follwoing dependencies:
+
+    - [PostgreSQL Setup Guide](https://github.com/gcivil-nyu-org/spring2020-cs-gy-9223-class/wiki/PostgreSQL-Setup-Guide)
+    - [Geckodriver Install Instruction](https://github.com/gcivil-nyu-org/spring2020-cs-gy-9223-class/wiki/Geckodriver---Install-instructions)
+    - [Grafana Setup Guide](https://github.com/gcivil-nyu-org/spring2020-cs-gy-9223-class/wiki/Grafana-Set-Up-Guide)
```

Comment:
Since we will move the repo to a new URL address, should we still use this wiki page link?

Reply:
This is a good point. It will be important to check every link on the new repo. But it only concerns us after the repo is transferred. Now the links in this PR work for now. And we don't have anything in the new repo yet, so it won't make sense to point to some empty pages that do not even exist at the moment.

Reply:
I tried to find methods to use the relative URL in markdown, but I failed to get a good solution. So let's just leave it as the current version.

Reply:
Sure. Let's keep it this way now.